keras_nlp/tokenizers/byte_pair_tokenizer.py

```diff
@@ -230,6 +230,7 @@ def __init__(
         vocabulary,
         merges,
         sequence_length=None,
+        add_prefix_space=False,
```

Comment:
Please add this argument to the doc-string. Can be something like this:

```
        add_prefix_space: boolean, defaults to False. If True, prepends a
            whitespace character to the input strings before tokenization. This
            is useful since the beginning of a word is detected by a leading
            space.```